packages/slate-react/src/components/editable.tsx

```diff
@@ -202,6 +204,32 @@ export const Editable = (props: EditableProps) => {
   const onDOMSelectionChange = useMemo(
     () =>
       throttle(() => {
+        const el = ReactEditor.toDOMNode(editor, editor)
+        const root = el.getRootNode()
+        const safariVersion = navigator.userAgent.match(/Version\/(\d+\.\d+)/)
+        const isSafariVersionLessThan17 =
```

Comment:
New browser version checks should live inside slate-react/src/utils/environment.ts so they're easy to find/consist/reused and then imported. Thanks.

Reply:
Updated in [c6dbf93](https://github.com/ianstormtaylor/slate/pull/5648/commits/c6dbf93a7ea1065a8eb44d02f94c1c77bd62e52e).